doc/news/interface_changes/README.rst

```diff
@@ -0,0 +1,9 @@
+Interface Changes
+=================
+
+Add news fragments for interface changes (e.g. xml changes) to this directory.
+The format should be ``<JIRA TICKET>.<COMPONENT>.<EXT>``, where ``<COMPONENT>`` is the name of the component in lowercase.
```

Comment:
You can use `towncrier create interface_changes/<JIRA_TICKET>.<COMPONENT>.<EXT>` as a shorthand.